### PR TITLE
Document #2540 - users need to add local CA cert to keystore

### PR DESCRIPTION
**Pull request by kolloch**

### CODE REVIEW

File: docs/docs/common-problems.md [hunk 1]
```diff
@@ -22,6 +27,29 @@ Once `hostname -f` returns successfully, Marathon should be able to start.
 
 The default executor timeout on mesos slaves or Marathon's `task_launch_timeout` are too low for Docker containerizers (docker pull can take some time...). Refer to the docs on [Docker Containers]({{ site.baseurl }}/docs/native-docker.html)
 
+# Using HTTPS with a local CA certificate
+
+You might get errors like the following in the Marathon log if you running Marathon with HTTPS-only
```
Comment (meichstedt) [inline docs/docs/common-problems.md]: s/if you running/if you are running

**Comment by meichstedt** [comment]
Please fix the typo and I'll merge!

**Comment by kolloch** [comment]
Fixed the error hopefully...

**Comment by meichstedt** [comment]
Yay!

**Pull request merged**